model/model.py

```diff
@@ -0,0 +1,73 @@
+import pickle
+import json
+
+import pandas as pd
+from sklearn.ensemble import RandomForestRegressor  # type: ignore [import-untyped]
+
+from data.stats.get_runway_stats import RunwayStats
+from data.stats.get_flight_stats import FlightStats
+from src.json_converter import DataConverter
+from src.data.types import Airport
+
+
+class Model:
+    def __init__(self) -> None:
+        self._model = None  # type: RandomForestRegressor
+
+    def load_trained_model(self, filename: str) -> None:
+        with open(filename, "rb") as f:
+            model = pickle.load(f)
+        if not isinstance(model, RandomForestRegressor):
+            raise AttributeError(
+                f"The file {filename!r} does not contain a valid model. "
+                f"Expected type {type(RandomForestRegressor)!r} but got {type(model)!r} "
+            )
+        self._model = model
+
+    def predict(self, prepped_input: pd.DataFrame) -> int:
+        return self._model.predict(prepped_input)
+
+    def train(self, data_filename: str) -> "Model":
+        with open(data_filename, "r", encoding="utf-8") as f:
+            raw_data = json.load(f)
+        target, features = self.preprocessing(raw_data)
+        return self.fit(features, target)
+
+    def fit(self, features: pd.DataFrame, target: pd.DataFrame) -> "Model":
+        self._model = RandomForestRegressor(n_estimators=10, max_features=2)
+        self._model.fit(features, target.values.ravel())
+        return self
+
+    def save_trained_model(self, filename: str) -> None:
+        with open(filename, "wb") as f:
+            pickle.dump(self._model, f)
+
+    def preprocessing(
+        self, raw_data: dict[str, list[Airport]]
+    ) -> tuple[pd.DataFrame, pd.DataFrame]:
```

Comment:
Could you move this to one line, it looks odd and inconsistent with everything else